### DISCUSSION TITLE
Preguntas Clase 2 - 5/23/2024

### DISCUSSION THREAD

**Discussion by DanteCely** [nodo-eafit-frontend/frontend-workshop-2024-1#3] [Q&A]
## Pregunten cualquier duda de la clase.

**Comment by MarceM03** [comment]
Marvin, WSL si puede ser instalado en un Mac? Por que no me salen opciones, incluso hay comentarios de que no existe para Mac.

**Reply by DanteCely** [reply]
Solo tienes que instalar otro terminal desde aquí 
https://iterm2.com/

**Comment by Cracklord98** [comment]
test

**Comment by JaimeGallo** [comment]
![Screenshot_20240523_193850_Code Studio](https://github.com/nodo-eafit-frontend/frontend-workshop-2024-f04/assets/170676691/c144a59f-bb9f-4c66-9687-2492c9bc23f1)
![Screenshot_20240523_193920_One UI Home](https://github.com/nodo-eafit-frontend/frontend-workshop-2024-f04/assets/170676691/e154c49d-db03-41aa-8d81-cee058a2b9c3)
Hola Marvin, adjunté la imagen de la aplicación de terminal para celular, se llama Code Studio. No sé si me pueda servir por el momento.

**Reply by DanteCely** [reply]
Mejor trabaja desde https://idx.dev/ mientrastanto

**Comment by RogerEscobar** [comment]
Hay partes especificas del reto que deben hacerse en terminal? como se entregaría eso especificamente?

**Reply by DanteCely** [reply]
Se entra a travez de los Pull REquest, en las siguientes Clases veeríamos los temas 😉

**Comment by diegolsg** [comment]
Profe cual es la extension de vsc para ver el navegador al lado y saber como se ven los cambios realizados

**Reply by DanteCely** [reply]
https://marketplace.visualstudio.com/items?itemName=ms-vscode.live-server

**Comment by vicoso27** [comment]
Profe tengo este problema con ubuntu 
![image](https://github.com/nodo-eafit-frontend/frontend-workshop-2024-f04/assets/106107164/4a45c0f0-a59f-49c5-a170-142fa3c72041)

**Reply by DanteCely** [reply]
Tendremos que buscar junto en internet para encontrar una solución.

```
WslRegisterDistribution failed with error: 0x8007019e
Error: 0x8007019e El subsistema de Windows para Linux no se habilitó
```
Buscando la respuesta del terminal en internet me topé con [esto](https://platzi.com/discusiones/2042-prework-windows/146181-hola-me-sale-este-error-como-lo-soluciono-gracias/), te puede servir. Confirmame si te sirvió y sigue buscando si encuentras la soluión, si no, lo buscamos juntos otra vez.

Parece que te falta correr un comando o un paso para habilitar subsitema de Windows

**Comment by TakUHG** [comment]
Como se abre esa ventana de la izquierda en visual studio code para empezar a trabajar con el css

**Reply by DanteCely** [reply]
Generalmente para trabajar con CSS tienes que crear un archivo CSS`styles.css` dentro de tu proyecto. Luego conectas el CSS usando <link /> en el HTML, ter recomiendo buscar la documentación en internet.
